zetaclient/bitcoin/bitcoin_client.go

```diff
@@ -38,7 +38,7 @@ import (
 )
 
 const (
-	DynamicDepositorFeeHeight = 832000 // Bitcoin block height to switch to dynamic depositor fee
+	DynamicDepositorFeeHeight = 834500 // Bitcoin block height to switch to dynamic depositor fee
```

Comment:
What's the context here? Do we need to update this value for every upgrade moving forward? Would need more comment what the height is used for.

Reply:
Only used once for next release. This height the starting height from which dynamic deposit fee will take effect for mainnet (only). I can update the comment.

Reply:
https://github.com/zeta-chain/node/pull/1818/commits/034f9ea7cf4b5928aaf1217aa5e888ed0214bdb8